services/headless-lms/migrations/20210916011528_add_supervisor_to_course_instances.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Add up migration script here
+ALTER TABLE course_instances
+ADD COLUMN contact_email VARCHAR(255),
+  ADD COLUMN supervisor_name VARCHAR(255),
+  ADD COLUMN supervisor_email VARCHAR(255);
```

Comment:
```suggestion
  ADD COLUMN teacher_in_charge_email VARCHAR(255);
```
+ CHECK condition that trims the string and checks that not empty